x-pack/plugins/apm/server/routes/mobile/get_mobile_location_stats.ts

```diff
@@ -0,0 +1,209 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  termQuery,
+  kqlQuery,
+  rangeQuery,
+} from '@kbn/observability-plugin/server';
+import { ProcessorEvent } from '@kbn/observability-plugin/common';
+import {
+  SERVICE_NAME,
+  SESSION_ID,
+  SERVICE_TARGET_TYPE,
+  CLIENT_GEO_COUNTRY_NAME,
+} from '../../../common/es_fields/apm';
+import { environmentQuery } from '../../../common/utils/environment_query';
+import { APMEventClient } from '../../lib/helpers/create_es_client/create_apm_event_client';
+import { getBucketSize } from '../../lib/helpers/get_bucket_size';
+import { getOffsetInMs } from '../../../common/utils/get_offset_in_ms';
+
+type Timeseries = Array<{ x: number; y: number }>;
+
+interface LocationStats {
+  mostSessions: {
+    location?: string | number;
```

Comment:
why are location and value as undefined when you return them always here
https://github.com/elastic/kibana/pull/149101/files#diff-f11b5173a4133cd4a82abcb14f61480e5336038b7cc4880eaa5654dc75fe8b00R136-R161 ?